products/elixir.md

```diff
@@ -10,39 +10,54 @@ activeSupportColumn: true
 releaseDateColumn: true
 sortReleasesBy: "release"
 releases:
+  - releaseCycle: "1.13"
+    release: 2021-12-03 # https://github.com/elixir-lang/elixir/releases/tag/v1.13.0
+    eol: 2024-06-01 # projected release date of 1.18.0
+    support: 2022-06-01 # projected release date of 1.14.0
+    latest: "1.13.2"
```

Comment:
are you sure is this correct ? it looks like it should be 1.13.1
https://github.com/elixir-lang/elixir/tags

Reply:
This one was confusing to me to! I initially picked 1.13.1 as the newest because of the tags (like you linked to).

Then, as I was confirming that links worked correctly in [the PR preview](https://deploy-preview-714--endoflife-date.netlify.app/elixir) I saw that the v1.13 CHANGELOG file lists 1.13.2 with a 2021-12-14 release date! https://github.com/elixir-lang/elixir/blob/v1.13/CHANGELOG.md#v1132-2021-12-14

My Homebrew only shows 1.13.1
```
$ brew info elixir   
elixir: stable 1.13.1 (bottled), HEAD
```

I don't feel strongly about this (nor have in-depth knowledge of the situation) so listing 1.13.1 as the latest to be on the safe side feels fine to me 🤷

Reply:
Switched back to listing 1.13.1 as the latest in f61cf8bede3e8cce096ed5cc716e57b3f1105477